src/TAction.h

```diff
@@ -81,9 +81,9 @@ class TAction : public Tree<TAction>, public QObject
     void setCommandButtonDown(const QString& cmd) { if(cmd != mCommandButtonDown) { setDataChanged(); mCommandButtonDown = cmd; } }
     QString getCommandButtonDown() { return mCommandButtonDown; }
     bool isPushDownButton() { return mIsPushDownButton; }
-    void setIsPushDownButton(bool b) { if(b != mIsPushDownButton) { setDataChanged(); mIsPushDownButton = b; } }
+    void setIsPushDownButton(bool button) { if(button != mIsPushDownButton) { setDataChanged(); mIsPushDownButton = button; } }
```

Comment:
The original `b` was probably intended to be short for "boolean" rather than "button":
```suggestion
    void setIsPushDownButton(bool isPushDown) { if (isPushDown != mIsPushDownButton) { setDataChanged(); mIsPushDownButton = isPushDown; } }
```